tests/integration.go

```diff
@@ -42,6 +42,7 @@ var (
 		"",
 		"Comma separated list of patterns to use to filter the envars to pass (set to \".*\" to pass everything)",
 	)
+	followRedirects = flag.Bool("client_follow_redirects", true, "Whether the test http client should follow http redirects or not")
```

Comment:
I'd set the flag the other way around using `client_not_follow_redirects` with the default value `false`, so the new client is introduced when the user adds the `client_not_follow_redirects` flag: